Essentials/src/main/java/com/earth2me/essentials/commands/Commandkill.java

```diff
@@ -29,12 +28,7 @@ protected void updatePlayer(final Server server, final CommandSource sender, fin
         if (sender.isPlayer() && user.isAuthorized("essentials.kill.exempt") && !ess.getUser(sender.getPlayer()).isAuthorized("essentials.kill.force")) {
             throw new PlayerExemptException("killExempt", matchPlayer.getDisplayName());
         }
-        final EntityDamageEvent ede = ess.getDamageEventProvider().callDamageEvent(matchPlayer, sender.isPlayer() && sender.getPlayer().getName().equals(matchPlayer.getName()) ? EntityDamageEvent.DamageCause.SUICIDE : EntityDamageEvent.DamageCause.CUSTOM, Float.MAX_VALUE);
```

Comment:
Removing the damage sources here and all the cancellation and permission checks below definitely isn't the way to go. Please see [my previous comment on the PR](https://github.com/EssentialsX/Essentials/pull/5850#issuecomment-2223132714).